lib/source/google_key_source.ex

```diff
@@ -26,4 +26,29 @@ defmodule ExFirebaseAuth.KeySource.Google do
     end)
     |> Enum.filter(fn {_, value} -> not is_nil(value) end)
   end
+
+  defp get_max_age(headers) do
+    headers |> get_cache_control_header() |> get_get_max_age_from_cache_control()
+  end
+
+  defp get_cache_control_header(headers) do
+    Enum.find(headers, fn {k, _v} -> k == "cache-control" end)
+  end
+
+  defp get_get_max_age_from_cache_control(nil) do
+    300
+  end
+
+  defp get_get_max_age_from_cache_control({"cache-control", value}) do
+    values = String.split(value, ",")
+
+    Enum.find_value(values, fn x ->
```

Comment:
`Enum.find_value/2` returns a default, which is currently nil but I think we should make that 300 as well (https://hexdocs.pm/elixir/Enum.html#find_value/3)

https://hexdocs.pm/elixir/Enum.html#find_value/3